.github/workflows/create-release-branch.yaml

```diff
@@ -36,29 +36,12 @@ jobs:
       if: ${{ !github.event.inputs.version }}
       run: echo "CURRENT_VERSION=$(echo ${{ steps.get-latest-tag.outputs.tag }} | sed 's/^gaffer2-//')" >> $GITHUB_ENV
 
-    - name: Bump latest tag variable
+    - name: Bump latest tag variable version
       if: ${{ !github.event.inputs.version }}
-      uses: actions-ecosystem/action-bump-semver@v1
-      id: bump-semver
-      with:
-        current_version: ${{ env.CURRENT_VERSION }}
-        level: minor
-
-    - name: Set version automatically
-      if: ${{ !github.event.inputs.version }}
-      run: echo "RELEASE_VERSION=$(echo ${{ steps.bump-semver.outputs.new_version }} )" >> $GITHUB_ENV
+      run: echo "RELEASE_VERSION=$(echo ${{ env.CURRENT_VERSION }} | sed -r 's/([0-9]+)\.([0-9]+)\.([0-9]+)/echo \1.$((\2+1)).\3/' | sh)" >> $GITHUB_ENV
```

Comment:
Does this sed command bump the minor version or patch version? I noticed it's the same as the hotfix command but the hotfix is meant to bump patch (2.0.0 -> 2.0.1) and release is meant to bump minor (2.0.0 -> 2.1.0)

Reply:
Yes this is a mistake, forgot to change to be the patch. I'll add a commit fixing this.